### PR TITLE
class02-hw3-JDKim

### PR DESCRIPTION
**Pull request by jd6286**
hw3 제출

### CODE REVIEW

File: class02/homework/JDKim/hw3/hw03_template.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+./splitted_dataset/train/<class#>: 6
+
+```
+
+## Training 결과
+
+| Classification model | Accuracy | FPS | Training time | Batch size | Learning rate | Other hyper-prams |
+| --- | --- | --- | --- | --- | --- | --- |
+| EfficientNet-V2-S | 1.0 |  | 0:00:28.887994 | 64 | 7.100e-03 |  |
+| MobileNet-V3-large-1x | 1.0 |  | 0:00:17.129582 | 64 | 1.0e-05 |  |
```
Comment (JerryJLee) [inline class02/homework/JDKim/hw3/hw03_template.md:24]: 2개 뿐인 이유라도? :)
